ghost/core/test/utils/fixtures/constants.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    OWNER_USER_ID: '5951f5fc0000000000000000'
```

Comment:
This is definitely a non-blocking concern, but I am wondering if we need to add this new system here. 

In the old test framework, we pull the hardcoded IDs out like this:

https://github.com/TryGhost/Ghost/blob/main/ghost/core/test/e2e-api/admin/actions.test.js#L96

In the new test framework we have a slightly nicer helper system for it:

https://github.com/TryGhost/Ghost/blob/main/ghost/core/test/e2e-api/admin/api-tokens.test.js#L163

What you've done here is clean and there's nothing wrong with it - but the question on my mind is are we going to pull out every hard-coded ID in the data-genrator and use it consistently, or is this just for the owner ID? In the latter case, I'd just have fixtures/owner-id.js rather than fixtures/constants.js.